Sources/ApolloCodegenLib/FileGenerators/OperationManifestFileGenerator.swift

```diff
@@ -14,7 +14,7 @@ struct OperationManifestItem {
 
     var source = operation.definition.source.convertedToSingleLine()
     for fragment in operation.referencedFragments {
-      source += #"\n\#(fragment.definition.source.convertedToSingleLine())"#
+      source += #" \#(fragment.definition.source.convertedToSingleLine())"#
```

Comment:
The recent change here was to make this a raw string with the `#` character for Swift. I think the root issue here is that because it is now a raw string the hash computation sees the raw two characters instead of when it was a regular string it would have been escaped down to an actual newline (single) character.

I don't know if it's 100% correct here to be replacing this with a string unless that's what we'll actually be sending - given that we're modifying the source it looks like it will be. Question then - does this break any backwards compatibility with older generated fragments or do we not need to care about that?

Reply:
That's a good point, the raw string could be contributing to the issue here. I would lean towards still removing the newline character as I don't remember there being any specific reason to have it other than we thought we should keep it before each fragment. This route would bring us inline with how Kotlin is handling this as well.

As for backwards compatibility I don't think this should be an issue because the original change was very recent for persisted queries and with this bug operations containing fragments aren't working correctly to begin with.

@AnthonyMDev thoughts?